tools/deb/istio-start.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Script to run Envoy for Istio.
+
+# Load config variables
+set -e
+
+ISTIO_SIDECAR_CONFIG=${ISTIO_SIDECAR_CONFIG:-/var/lib/istio/envoy/sidecar.env}
+
+if [[! -f $ISTIO_SIDECAR_CONFIG]]; then
+  echo "Missing sidecar config: $ISTIO_SIDECAR_CONFIG"
+  exit 1
+fi
+
+. $ISTIO_SIDECAR_CONFIG
+
+/usr/local/bin/istio-iptables.sh
+
+
+if [ -f /usr/local/bin/pilot-agent ]; then
+  exec su - istio /usr/local/bin/pilot-agent proxy > /var/log/istio/istio.log 2>&1
+else
+  if [ "" = "$SVC_IP" ]; then
+    SVC_IP=$(hostname --ip-address)
+  fi
+  # Run envoy directly - agent not installed. This should be used only for debugging/testing standalone envoy
+  exec su - istio -c "/usr/local/bin/envoy -c /var/lib/istio/envoy/envoy.json --restart-epoch 0 --drain-time-s 2 --parent-shutdown-time-s 3 --service-cluster istio-proxy --service-node '$SVC_IP|mysvc.default|cluster.local' $ISTIO_DEBUG"
+fi
```

Comment:
I kind of tried all 3 options - if/when we switch to 'repo' or other 'unified' build it won't matter. But in the current bazel world - with manual SHA copy and total mess, I think proxy is the best place - that's where most of the sidecar is built.
It's also what would be expected with bazel - you build envoy in this repo, it should be packaged here too, same as we for docker.

One thing I'm conflicted about is the pilot-agent - right now I'm creating a separate .deb ( for same reasons - should be packaged where it is built, as would be expected ), k8s also has one deb per component. However it would be 
easier to make an exception and pull the agent into proxy.  

Can we iterate - I don't want to bike shed the location decision in the PR review :-)